src/utils/parsers/common.ts

```diff
@@ -163,7 +163,7 @@ export abstract class BaseParser {
         }
 
         return {
-            content: lines.reduce((prev, curr) => prev + curr.content, ''),
+            content: lines.reduce((prev, curr) => prev + (prev ? '\n' : '') + curr.content, ''),
```

Comment:
could you not just use `lines.join('\n')` here?

Reply:
the lines  array contains `LineItem` objects, not strings unfortunately

Reply:
What if we get the content by using the start/end lines instead? Since this is using the filtered file it will be missing the comment lines. Instead we could say `content = file.lines.slice(start - 1, end).map(ln => ln.content).join('\n')`

Reply:
That would also make it easy to add surrounding lines later

Reply:
i changed it to just `lines.map(line => line.content).join('\n')`. It's simpler but also doesn't need to check the file for the lines since it already has the lines it needs to display